app/controllers/api/v2/registration_commands_controller.rb

```diff
@@ -17,7 +17,7 @@ class RegistrationCommandsController < V2::BaseController
         param :setup_remote_execution, :bool, desc: N_("Set 'host_registration_remote_execution' parameter for the host. If it is set to true, SSH keys will be installed on the host")
         param :jwt_expiration, :number, desc: N_("Expiration of the authorization token (in hours)")
         param :insecure, :bool, desc: N_("Enable insecure argument for the initial curl")
-        param :packages, String, desc: N_("Packages to install on the host when registered. Can be set by `host_packages` parameter. Example: `pkg1 pkg2`")
+        param :packages, String, desc: N_("Packages to install on the host when registered. Can be set by `host_packages` parameter, example: `pkg1 pkg2`.")
```

Comment:
Well, that's my mistake I didn't notice, but we avoid the dots at the end of param descriptions. At least for the sake of consistency.